xbmc/windowing/X11/CMakeLists.txt

```diff
@@ -1,23 +1,33 @@
-set(SOURCES GLContextEGL.cpp
-            GLContextGLX.cpp
-            GLContext.cpp
-            WinEventsX11.cpp
-            WinSystemX11.cpp
+set(SOURCES GLContext.cpp
             WinSystemX11GLContext.cpp
-            WinSystemX11GLESContext.cpp
-            XRandR.cpp
-            VideoSyncGLX.cpp
-            VideoSyncDRM.cpp)
+            WinSystemX11.cpp
+            WinEventsX11.cpp)
 
 set(HEADERS GLContext.h
-            GLContextEGL.h
-            GLContextGLX.h
-            WinEventsX11.h
-            WinSystemX11.h
             WinSystemX11GLContext.h
-            WinSystemX11GLESContext.h
-            XRandR.h
-            VideoSyncGLX.h
-            VideoSyncDRM.h)
+            WinSystemX11.h
+            WinEventsX11.h)
+
+if(EGL_FOUND)
```

Comment:
I guess it doesn't really make much sense to break it up as all of the sources are required.

Would it make sense to have the ability to disable glx? This would require minimal ifdefs